extensions/python/src/main/java/com/hazelcast/jet/python/PythonServiceContext.java

```diff
@@ -108,6 +109,14 @@
         }
     }
 
+    private void checkIfPythonIsAvailable() {
+        try {
+            new ProcessBuilder("python3", "--version").start().waitFor();
```

Comment:
Is there a minimal python3 version we require?

Also, this will pass if `python3 --version` runs, but returns exit code other than 0. Maybe it would be better to log the output of the command, exit code. Possibly parse the version & check.